devtools-ci-index.yaml

```diff
@@ -680,7 +680,7 @@
             ssh_cmd="ssh $sshopts $CICO_hostname"
             $ssh_cmd yum -y install rsync
             rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload
-            $ssh_cmd -t "cd payload && /bin/bash cico_run_EE_tests.sh $EE_TEST_USERNAME $EE_TEST_PASSWORD"
+            $ssh_cmd -t "cd payload && /bin/bash cico_run_EE_tests.sh"
```

Comment:
rather than pass the param's from script to script, its now just in the ENV and can be consumed that way